web/react/utils/client.jsx

```diff
@@ -279,24 +279,33 @@ module.exports.getAudits = function(userId, success, error) {
     });
 };
 
-module.exports.getMe = function(success, error) {
+module.exports.getMeSynchronous = function(success, error) {
+    var currentUser = null;
 
     $.ajax({
+        async: false,
```

Comment:
This probably isn't the PR to change this in but just want to point out from #370 that the `async: false` setting is deprecated by jquery so we should probably find another way to do this if possible.